src/algorithms/wolfe_line_search.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <RcppArmadillo.h>
+#include "../families/family.h"
+
+using namespace Rcpp;
+using namespace arma;
+
+template <typename T>
+double Family::zoom(const T& x, const mat&y, const double rho, const mat&u, 
+                    const mat& z, const mat& d, double t_low, double t_high){
```

Comment:
If argument lists are too long to fit one one line, put every argument on its own line.

```suggestion
double Family::zoom(const T& x, 
                    const mat& y, 
                    const double rho, 
                    const mat& u, 
                    const mat& z, 
                    const mat& d, 
                    double t_low, 
                    double t_high)
{
```